internal/server/multitherading.go

```diff
@@ -75,7 +75,7 @@ func (s *AsyncServer) gather(redisCmd *cmd.RedisCmd, buf *bytes.Buffer, numShard
 	switch c {
 	case SingleShard, Custom:
 		if evalResp[0].Error != nil {
-			buf.Write([]byte(evalResp[0].Error.Error()))
+			buf.WriteString(evalResp[0].Error.Error())
```

Comment:
I am not sure whether the string will replace the characters \r\n hence kept it as a byte slice. But if you can check an error scenario then it would be great.

Reply:
Was a recommendation from lint, have reverted it for now.